repose-aggregator/components/filters/rate-limiting-filter/pom.xml

```diff
@@ -130,6 +130,20 @@
             <artifactId>scala-library</artifactId>
         </dependency>
 
+        <!-- LogFactory, needed by MockHttpServletRequest/Response -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <!-- MockHttpServletRequest/Response -->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Oh, yeah.  The gradle build wasn't a thing when I initially worked this story.  Since the build isn't working at the moment, I'm not sure if I should add the lines I expect would work without testing it or hold off and let the gradle work take care of it.